src/safe_cli/operators/safe_operator.py

```diff
@@ -936,7 +943,7 @@ def execute_safe_transaction(self, safe_tx: SafeTx):
             else:
                 call_result = safe_tx.call(self.hw_wallet_manager.sender.address)
             print_formatted_text(HTML(f"Result: <ansigreen>{call_result}</ansigreen>"))
-            if yes_or_no_question("Do you want to execute tx " + str(safe_tx)):
+            if self._is_confirmed_transaction_execution(safe_tx):
```

Comment:
nitpick: I find that extracting this to a function to be more confusing. I think it's clearer if the expression is in the `if` itself:

```python
if self.script_mode or yes_or_no_question(f"Do you want to execute tx {safe_tx}"):
```

Reply:
An alternative would be to try to embed the `script_mode` directly in the `yes_or_no_question` as then we would cover all use cases. If we miss one `yes_or_no_question` this might block the script.

Reply:
Done [here](https://github.com/safe-global/safe-cli/pull/419/commits/1e1d3548bd4c283e2de19261d3936d82d86bf698).